### ISSUE TITLE
Allow events to access data loaded by page/layout.

### ISSUE THREAD

**Issue by jnordberg** [sveltejs/kit#9411]
### Describe the problem

In an event handler there is no way to call `const myData = await parent()`

### Describe the proposed solution

The `parent` function could be made accessible in the scope of an event handler 

### Importance

would make my life easier

### Additional Information

Workaround is using hooks to populate locals with the data needed

**Comment by Conduitry** [comment]
What are you trying to do? You can access the `page` store and its `data` property from anywhere in code running on the browser, including in event handlers.

**Comment by jnordberg** [comment]
Sorry, I meant, inside [form action handlers](https://kit.svelte.dev/docs/form-actions) (I updated the issue to reflect this)

**Comment by dummdidumm** [comment]
What's a use case where this is needed / where adding the required data to `event.locals` is not enough or too cumbersome?

**Comment by jnordberg** [comment]
When you have many routes and only want data populated on some of them. Same use cases as loading stuff in layout.server/page.server I guess

**Comment by dummdidumm** [comment]
But which data of the loaders do you need inside the form action that you can't also get through either passing the data through the form or through `event.locals`?

**Comment by jnordberg** [comment]
Yes I can pass it in event.locals but that negates the whole idea behind load and parent, no?

If you want a concrete example I started implementing user session loading ([as suggested by the documentation](https://kit.svelte.dev/docs/load#cookies-and-headers)) in a admin/+layout.server.ts route but I had to refactor that to a hook that instead injected it globally in locals once I tried to use form actions
